package.json

```diff
@@ -494,20 +496,20 @@
     "dependencies": {
         "@azure/arm-appservice": "^11.0.0",
         "@azure/arm-resources": "^5.0.0",
-        "@microsoft/vscode-azext-azureappservice": "^0.6.4",
-        "@microsoft/vscode-azext-azureutils": "^0.3.4",
-        "@microsoft/vscode-azext-utils": "^0.3.21",
+        "@microsoft/vscode-azext-azureutils": "0.4.0-alpha.1",
+        "@microsoft/vscode-azext-utils": "0.4.3-alpha",
+        "@microsoft/vscode-azureresources-api": "^2.0.3",
         "@octokit/rest": "^18.5.2",
+        "buffer": "^6.0.3",
         "dayjs": "^1.11.0",
         "git-url-parse": "^13.1.0",
         "open": "^8.0.4",
         "semver": "^7.3.5",
         "vscode-nls": "^4.1.1",
         "vscode-uri": "^3.0.2",
-        "yaml": "^1.10.2"
+        "yaml": "^2.2.1"
     },
     "extensionDependencies": [
-        "ms-vscode.azure-account",
```

Comment:
I need to figure out how to have a separate package.json to have different extensionDependencies for .dev and desktop

Reply:
I thought we determined this wasn't possible, did something change?

Reply:
I have a feeling that there has to be someway to do it if they recommend that you don't 🤔

Reply:
> We do not recommend that extensions ship a different package.json for the web platform or to remove parts of the VSIX that do not work in the web.

Can't we just turn Azure Account into a soft-hard dependency?

Reply:
Or potentially, just auto-install it after activation when on desktop (if it's not present already).

Reply:
> Or potentially, just auto-install it after activation when on desktop (if it's not present already).

Oh right, I forgot we could do that.

Reply:
> Or potentially, just auto-install it after activation when on desktop (if it's not present already).

I'm okay to do that. It's a slightly worse user-experience, but not that bad. Also, we can include (if it's not already) accounts in the Azure extension pack.

Reply:
I figured there's no point in having Accounts as a dependency here since Resources is already a dependency.

Reply:
Agreed